torchdynamo/debug_utils.py

```diff
@@ -159,32 +176,70 @@ def inductor_fails(fx_g, args, check_str=None):
     return False
 
 
-def dump_to_minify(gm, args):
+def nvfuser_fails(fx_g, args, check_str=None):
+    from torch.fx.passes.backends.nvfuser import NvFuserBackend
+
+    nvfuser = NvFuserBackend()
+
+    try:
+        compile_mod = nvfuser(fx_g, args)
+        compile_mod = compile_mod(*args)
```

Comment:
If args is a single tensor will this unpack it?

Reply:
Similar to above, it's an invariant that args is always a tuple of tensors.